src/main/java/pl/piekoszek/gorskimatches/GorskiMatchesApplication.java

```diff
@@ -3,11 +3,15 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import javax.mail.MessagingException;
+
 @SpringBootApplication
 public class GorskiMatchesApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(GorskiMatchesApplication.class, args);
-	}
+    public GorskiMatchesApplication() {
```

Comment:
why?

Reply:
Oh, I think I was testing out the email.